src/crdt_bcounter.erl

```diff
@@ -154,6 +155,28 @@ to_binary(C) -> term_to_binary(C).
 -spec from_binary(binary()) -> bcounter().
 from_binary(<<B/binary>>) -> binary_to_term(B).
 
+%% @doc The following operation verifies
+%%      that Operation is supported by this particular CRDT.
+-spec is_operation(term()) -> boolean().
+is_operation(Operation) ->
```

Comment:
Please use andalso/2 instead of and/2 for short-circuiting here (and in the other type checks)!